clang/include/clang/Tooling/DependencyScanning/ModuleDepCollector.h

```diff
@@ -279,13 +281,16 @@ class ModuleDepCollector final : public DependencyCollector {
   /// Add module files (pcm) to the invocation, if needed.
   void addModuleFiles(CompilerInvocation &CI,
                       ArrayRef<ModuleID> ClangModuleDeps) const;
+  void addModuleFiles(CowCompilerInvocation &CI,
```

Comment:
Where are we using the non-Cow version of this function?

Reply:
That's used from `ModuleDepCollector::applyDiscoveredDependencies()` on the `-cc1` invocations expanded from the TU driver invocation.